cvmfs/cvmfs.cc

```diff
@@ -1270,6 +1270,33 @@ static void cvmfs_statfs(fuse_req_t req, fuse_ino_t ino) {
   fuse_reply_statfs(req, &info);
 }
 
+const uint64_t kMaxMetainfoLength = 64*1024;
+
+string GetRepoMetainfo(uint64_t size) {
+  if (!mount_point_->catalog_mgr()->manifest()) {
+    return "Manifest not available";
+  }
+
+  shash::Any hash = mount_point_->catalog_mgr()->manifest()->meta_info();
```

Comment:
This hash can be the Null hash (`hash.IsNull() == true`).  In this case, the result should be "Metainfo not available"